Makefile

```diff
@@ -153,7 +153,7 @@ dbus-common-install:
 	install -d $(DESTDIR)/etc/dbus-1/system.d
 	install -d $(DESTDIR)/$(INSTALL_DIR)/dbus-1/system-services
 	install -d $(DESTDIR)/$(LIBEXEC_DIR)
-	install -d $(DESTDIR)/etc/bash_completion.d
```

Comment:
At least for SLES11 support we need to install to /etc/bash_completion.d. I'd suggest we conditionally do that.

Reply:
You're right. I'll work on a solution, and squash my commits. You hear from me.
Thank you ;)